app/views/setup/welcome.html.slim

```diff
@@ -156,5 +156,20 @@ h1 Initial CaaS Platform Configuration
   = render partial: 'setup/cloud/settings', locals: { f: f }
   = render partial: 'setup/settings/container_runtime', locals: { f: f }
 
+  .panel.panel-default
```

Comment:
To avoid growing this file more and more, extract this to a partial like we did for some of the other configs.